src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestAddon.ts

```diff
@@ -156,8 +156,7 @@ export class SuggestAddon extends Disposable implements ITerminalAddon, ISuggest
 			doNotRequestExtensionCompletions = true;
 		}
 
-		const enableExtensionCompletions = this._configurationService.getValue<ITerminalSuggestConfiguration>(terminalSuggestConfigSection).enableExtensionCompletions;
-		if (enableExtensionCompletions && !doNotRequestExtensionCompletions) {
+		if (!doNotRequestExtensionCompletions) {
```

Comment:
before, for the tests, we were passing in `false` for extension completion enablement. without this, it now goes in here, causing the tests to fail.